internal/v1/server.go

```diff
@@ -413,13 +413,13 @@ func (h *Handlers) ComposeImage(ctx echo.Context) error {
 		return err
 	}
 
-	quotaOk, err := common.CheckQuota(idHeader.Identity.AccountNumber, h.server.db, h.server.quotaFile)
-	if err != nil {
-		return err
-	}
-	if !quotaOk {
-		return echo.NewHTTPError(http.StatusForbidden, "Quota exceeded for user")
-	}
+	// quotaOk, err := common.CheckQuota(idHeader.Identity.AccountNumber, h.server.db, h.server.quotaFile)
```

Comment:
I think this is not intentional, right?

Reply:
Definitely not! Had to comment that out for it to run locally. Will fix that now